forms/api/payment.php

```
@@ -0,0 +1,158 @@
+                $order['id'],
+                $charge['id'],
+                $payment_status,
+                (float) $order['amount'],
+                Currency::from($order['currency']),
+                PaymentType::from($input['payment_type']),
+                $charge['source']['type'],
+                $user_message,
+                json_encode($charge),
+            );
+
+            update_and_fetch_order_for_payment(
```

PHPStan (2.1.46)

[warning] Function update_and_fetch_order_for_payment invoked with 4 parameters, 3 required.


(arguments.count)